extensions/amp-fx-scroll-sync/0.1/scroll-sync-service.js

```diff
@@ -106,7 +110,7 @@ function onScroll(state) {
     // If last scroll is a LOT different than current scroll.
     // Do unsync transition.
   }
-  lastScrollTop_ = scrollTop;
+  this.lastScrollTop_ = scrollTop;
```

Comment:
This wouldn't work. `this` here is the `window` context and not the service.

If you want just drop `lastScrollTop_` for now since we're not using it yet.